### PR TITLE
Add PaddlePaddle guest blog post

### PR DESCRIPTION
**Pull request by sijunhe**

### CODE REVIEW

File: paddlepaddle.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Welcome PaddlePaddle to the Hugging Face Hub" 
+thumbnail: /blog/assets/126_paddlepaddle/thumbnail.jpg
+---
+
```
Comment (osanseviero) [inline paddlepaddle.md:5]: For all assets except the thumbnail, we try to have them in https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog

File: paddlepaddle.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# Welcome PaddlePaddle to the Hugging Face Hub
+
+<div class="blog-metadata">
+    <small>Published <TBD> 2023.</small>
```
Comment (osanseviero) [inline paddlepaddle.md:9]: TODO before merge

File: paddlepaddle.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+---
```
Comment (osanseviero) [inline paddlepaddle.md:1]: You need to update https://github.com/huggingface/blog/blob/main/_blog.yml as well

File: paddlepaddle.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1654942635336-5f3ff69679c1ba4c353d0c5a.png?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>PaddlePaddle</code>
+            <span class="fullname">PaddlePaddle</span>
+            <span class="bg-gray-100 dark:bg-gray-700 rounded px-1 text-gray-600 text-sm font-mono">guest</span>
+        </div>
+    </a>
+</div>
+
+We are happy to share an open-source partnership between HuggingFace and [PaddlePaddle](https://www.paddlepaddle.org.cn/en) on a shared mission to advance and democratize AI through open source!
```
Comment (osanseviero) [inline paddlepaddle.md:26]:
```suggestion
We are happy to share an open-source collaboration between Hugging Face and [PaddlePaddle](https://www.paddlepaddle.org.cn/en) on a shared mission to advance and democratize AI through open source!
```

File: paddlepaddle.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+![thumbnail](assets/126_paddlepaddle/thumbnail.jpg)
+
+**With [PaddleNLP](https://huggingface.co/docs/hub/paddlenlp) leading the way, PaddlePaddle libraries will gradually integrate with the HuggingFace Hub.** You will soon be able to play with the full suite of awesome pre-trained PaddlePaddle models across text, image, audio, video and multi-modalities on the Hub!
```
Comment (osanseviero) [inline paddlepaddle.md:32]:
```suggestion
**With [PaddleNLP](https://huggingface.co/docs/hub/paddlenlp) leading the way, PaddlePaddle libraries will gradually integrated with the Hugging Face Hub.** You will soon be able to play with the full suite of awesome pre-trained PaddlePaddle models across text, image, audio, video and multi-modalities on the Hub!
```

File: paddlepaddle.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+## Find PaddlePaddle Models
+
+You can find all PaddlePaddle models on the Model Hub by with the [PaddlePaddle library tag](https://huggingface.co/models?library=paddlepaddle).
```
Comment (osanseviero) [inline paddlepaddle.md:36]:
```suggestion
You can find all PaddlePaddle models on the Model Hub by filtering with the [PaddlePaddle library tag](https://huggingface.co/models?library=paddlepaddle). 
```

**Review by osanseviero — COMMENTED** [review]
Initial high-level pass. Let us know when this PR is not a draft and is ready for full review

File: paddlepaddle.md (hunk 5 above)
Comment (sijunhe) [reply paddlepaddle.md:32]: I think this should be `integrate` here since it's a verb and comes after "will"

File: paddlepaddle.md (hunk 1 above)
Comment (sijunhe) [reply paddlepaddle.md:5]: Sure. Do i just submit a PR to the repo?

File: paddlepaddle.md (hunk 5 above)
Comment (osanseviero) [reply paddlepaddle.md]: WDYT of this update?

Comment (sijunhe) [reply paddlepaddle.md]: Grammarly says "passive voice misuse". How about "With PaddleNLP leading the way, PaddlePaddle will gradually integrate its libraries with the Hugging Face Hub"

File: paddlepaddle.md (hunk 1 above)
Comment (sijunhe) [reply paddlepaddle.md:5]: https://huggingface.co/datasets/huggingface/documentation-images/discussions/30

Comment (osanseviero) [reply paddlepaddle.md:5]: Merged!

File: paddlepaddle.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+## Find PaddlePaddle Models
+
+You can find all PaddlePaddle models on the Model Hub by filtering with the [PaddlePaddle library tag](https://huggingface.co/models?library=paddlepaddle). 
+
+<p align="center">
+  <img src="assets/126_paddlepaddle/paddle_tag.png" alt="PaddlePaddle Tag"/>
+</p>
+
+There is already over [Numbers TBD] PaddlePaddle models on the Hub. As an example, you can find our multi-task Information Extraction model series [UIE](https://huggingface.co/PaddlePaddle/uie-base), State-of-the-Art Chinese Language Model [ERNIE 3.0 model series](https://huggingface.co/PaddlePaddle/ernie-3.0-nano-zh), novel document pre-training model [Ernie-Layout](PaddlePaddle/ernie-layoutx-base-uncased) with layout knowledge enhancement in the whole workflow and so on.
```
Comment (osanseviero) [inline paddlepaddle.md:42]: To update

File: paddlepaddle.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1654942635336-5f3ff69679c1ba4c353d0c5a.png?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>PaddlePaddle</code>
+            <span class="fullname">PaddlePaddle</span>
+            <span class="bg-gray-100 dark:bg-gray-700 rounded px-1 text-gray-600 text-sm font-mono">guest</span>
+        </div>
+    </a>
+</div>
+
+We are happy to share an open-source collaboration between Hugging Face and [PaddlePaddle](https://www.paddlepaddle.org.cn/en) on a shared mission to advance and democratize AI through open source!
```
[the remark this answers is not in the document]
Comment (sijunhe) [reply paddlepaddle.md]: Changed all but the last "open-sourcing"

File: paddlepaddle.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+## Find PaddlePaddle Models
+
+You can find all PaddlePaddle models on the Model Hub by filtering with the [PaddlePaddle library tag](https://huggingface.co/models?library=paddlepaddle). 
+
+<p align="center">
+  <img src="assets/126_paddlepaddle/paddle_tag.png" alt="PaddlePaddle Tag"/>
+</p>
+
+There is already over 75 PaddlePaddle models on the Hub. As an example, you can find our multi-task Information Extraction model series [UIE](https://huggingface.co/PaddlePaddle/uie-base), State-of-the-Art Chinese Language Model [ERNIE 3.0 model series](https://huggingface.co/PaddlePaddle/ernie-3.0-nano-zh), novel document pre-training model [Ernie-Layout](PaddlePaddle/ernie-layoutx-base-uncased) with layout knowledge enhancement in the whole workflow and so on.
```
Comment (xianbaoqian) [inline paddlepaddle.md:42]: Nit: There are

**Review by xianbaoqian — APPROVED** [review]

**Pull request merged**